plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -734,366 +739,372 @@ def get_want_authentication_policy_server(self, auth_policy_server):
         Check the return value of the API with check_return_status()
 
         Parameters:
-            auth_policy_server (dict) - Playbook authentication policy server details
+            auth_policy_server (list of dict) - Playbook authentication policy server details
             containing IpAddress, authentication port, accounting port, Cisco ISE Details,
             protocol, retries, role, timeout seconds, encryption details.
 
         Returns:
             self - The current object with updated desired Authentication Policy Server information.
         """
 
-        auth_server = {}
-        auth_server_exists = self.have.get("authenticationPolicyServer").get("exists")
-        auth_server_details = self.have.get("authenticationPolicyServer").get("details")
-        trusted_server = False
-        if not auth_server_exists:
-            server_type = auth_policy_server.get("server_type")
-            if server_type not in ["ISE", "AAA", None]:
-                self.msg = "The server_type should either be ISE or AAA but not {0}.".format(server_type)
-                self.status = "failed"
-                return self
+        all_auth_server_details = []
+        auth_server_index = 0
+        for item in auth_policy_server:
+            auth_server = {}
+            auth_server_exists = self.have.get("authenticationPolicyServer")[auth_server_index].get("exists")
+            auth_server_details = self.have.get("authenticationPolicyServer")[auth_server_index].get("details")
+            trusted_server = False
+            if not auth_server_exists:
+                server_type = item.get("server_type")
+                if server_type not in ["ISE", "AAA", None]:
+                    self.msg = "The server_type should either be ISE or AAA but not {0}.".format(server_type)
```

Comment:
'server_type'

            self.msg = "The server_type should either be 'ISE' or 'AAA', but '{0}' was provided.".format(server_type)